types.go

```diff
@@ -0,0 +1,123 @@
+package gosdk
```

Comment:
cross check this with types in internal codebase as the example repo I used might be outdated